tests/baselines/reference/user/webpack.log

```diff
@@ -1,7 +1,217 @@
 Exit Code: 2
 Standard output:
-lib/ExportsInfo.js(949,28): error TS1016: A required parameter cannot follow an optional parameter.
-lib/dependencies/HarmonyExportImportedSpecifierDependency.js(94,37): error TS1016: A required parameter cannot follow an optional parameter.
+bin/webpack.js(63,8): error TS2300: Duplicate identifier 'path'.
+bin/webpack.js(127,8): error TS2300: Duplicate identifier 'path'.
+lib/ChunkGraph.js(500,3): error TS2741: Property 'prototype' is missing in type 'RuntimeSpecSet' but required in type 'typeof RuntimeSpecSet'.
+lib/ChunkGraph.js(1160,41): error TS2339: Property 'values' does not exist on type 'typeof RuntimeSpecMap'.
```

Comment:
This is because `module.exports.K = K` doesn't create an alias like `export { K }` does. Filed #40155 to track this, where the real fix is probably to make at least the previous construction create an export alias.